client/clientservice/src/request_service.cpp

```diff
@@ -68,10 +71,12 @@ void RequestServiceCallData::sendToConcordClient() {
   bool is_any_request_type = false;
   bft::client::Msg msg;
   if (request_.has_typed_request()) {
-    google::protobuf::Any app_request = request_.typed_request();
-    size_t request_size = app_request.ByteSizeLong();
+    ConcordClientRequest concordRequest;
+    concordRequest.set_client_service_id(client_->getSubscriptionId());
+    concordRequest.mutable_application_request()->CopyFrom(request_.typed_request());
+    size_t request_size = concordRequest.ByteSizeLong();
     std::string request(request_size, '\0');
-    app_request.SerializeToArray(request.data(), request_size);
+    concordRequest.SerializeToArray(request.data(), request_size);
     msg = bft::client::Msg(request.begin(), request.end());
     is_any_request_type = true;
   } else {
```

Comment:
What's the benefit of not wrapping the raw requests sent by the client as well?
I.e., this way we cannot consistently handle all types of requests coming via `RequestService` by the replicas, as they have to figure out their type (i.e., they cannot assume all requests are a `ConcordClientRequest`).

Reply:
This will require some more changes when we process the request and so will be taken up later.